### PR TITLE
chore(JUGO-3352): add github workflow to publish to packages, fix api…

### PR DESCRIPTION
**Pull request by joshrichards37**
… call to fetch shedule timeline

### CODE REVIEW

File: lib/restApi.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ function getPathWithIdentifier(path, params, actionPath) {
     pathParams = params;
   }
 
-  return params ? path + pathParams : path;
+  return params ? path + pathParams + actPath : path;
```
Comment (lintong) [inline lib/restApi.js:66]: is actPath won't render if PathParams is null? Is this desired behaviour?

Comment (joshrichards37) [reply lib/restApi.js:66]: yeah, if params is null then it shouldnt do anything, it means its got bad input. this repo needs a lot of love and attention, wanna change it to typescript as well

**Comment by jugory** [comment]
I've got a workflow with semantic versioning you can consider to use https://github.com/jugo-io/react-pixel-streaming/blob/main/.github/workflows/publish.yml

**Comment by joshrichards37** [comment]
> I've got a workflow with semantic versioning you can consider to use https://github.com/jugo-io/react-pixel-streaming/blob/main/.github/workflows/publish.yml

cheers mate, updated to use this workflow

**Review by lintong — APPROVED** [review]

**Pull request merged**